src/Compilers/CSharp/Test/Emit/CodeGen/ForeachTest.cs

```diff
@@ -441,36 +441,132 @@ public static void Main()
 
 ", TestOptions.ReleaseExe);
 
-            CompileAndVerify(comp, expectedOutput: "123").VerifyIL("Test.Main", @"
+            CompileAndVerify(comp, expectedOutput: "RedGreenBlue", verify: Verification.Fails).VerifyIL("Test.Main", @"
```

Comment:
Please add a comment for Verification.Fails so we can follow up. Maybe create an issue too?

Reply:
We are using an unsafe Span constructor here that takes a pointer. This is by design. I will add a comment.  

---
In reply to: [171432198](https://github.com/dotnet/roslyn/pull/24621#discussion_r171432198) [](ancestors = 171432198)